addons/udes_delivery_control/models/picking_extra_fields.py

```diff
@@ -114,6 +119,22 @@ def _compute_backload_time_taken(self):
                     record.backload_start_date, record.backload_end_date
                 )
 
+    @api.depends("loading_type")
+    # @api.onchange("u_loading_type")
+    def _compute_loading_type(self):
+        """Compute is_backloading and is_unloading from loading_type"""
+        # TODO improve the code
+        for record in self:
```

Comment:
Using Adam's values for `loading_type` we could simplify this to:
```
for record in self:
    record.is_unload = "unload" in record.loading_type
    record.is_backload = "backload" in record.loading_type
```

Reply:
done!